angularjs-portal-frame/src/main/webapp/bower_components/uw-ui-toolkit/src/less/uw-ui-toolkit/my-uw/uw-myuw-portlet.less

```diff
@@ -1,5 +1,103 @@
 /* MyUW Portlet Styles */
 
+.my-uw {
+  /* Buttons */
+  .btn-lg {
+    padding:12px 40px;
+  }
+  .btn-sm {
+    padding:4px 12px;
+  }
+  .btn-xs {
+    padding:2px 5px;
+  }
+  .btn {
+    background-color: #b70101;
+    margin: 10px 0px;
+    color: #fff;
+    border-radius: 4px;
+    border: 0px solid transparent;
+    box-shadow: 0px 5px 0px #600;
+    cursor: pointer;
+    transition: 0.1s ease-in-out;
+    position: relative;
+  }
+  .btn:hover {
+    transform: translateY(2px);
+    background-color:#900;
+    text-decoration:none;
+    box-shadow:0px 3px 0px #000;
+    transition:0.1s ease-in-out;
+  }
+  // .btn:active {
+  //   box-shadow:0px 1px 0px #000;
+  //   transform:translateY(4px);
+  // }
```

Comment:
This is fine, of course.

That said, you might find that you can worry less about needing to bring things back in the future what with `git` being awesome, *especially* if you make changes in smaller commits so you can more easily search the history to find that change.

```
Removed .btn:active
```

on a commit removing that would make it oh so easy to search by `.btn:active` or even by `.btn` and find that commit -- and have that commit contain precisely the change that, reversed, would re-add the style.  :smile:

[`#embraceChange`](http://books.google.com/books/about/Extreme_Programming_Explained.html?id=G8EL4H4vf7UC).